save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/vulnerability/VulnerabilityProjectDto.kt

```diff
@@ -0,0 +1,15 @@
+package com.saveourtool.save.entities.vulnerability
+
+import kotlinx.serialization.Serializable
+
+/**
+ * @property name of project
+ * @property url of project
+ * @property isOpen source project
+ */
+@Serializable
+data class VulnerabilityProjectDto(
+    val name: String,
+    val url: String,
+    val isOpen: Boolean,
```

Comment:
It should be `isOpenSource` then, shouldn't it?

```suggestion
    val isOpenSource: Boolean,
```

Reply:
Fixed